src/AvTranscoder/data/decoded/VideoFrame.cpp

```diff
@@ -76,9 +79,6 @@ size_t VideoFrame::getSize() const
 
 void VideoFrame::allocateData()
 {
-    if(_dataAllocated)
-        return;
```

Comment:
Removing this, you allow an allocated frame to be reallocated, is that really what you want?